samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -71,12 +71,27 @@
     /* "Key": // dotnet user-secrets set "AzureSpeech:Key" "MY_AZURE_SPEECH_KEY" */
   },
 
+  "Auth": {
```

Comment:
[nit] Avoid unnecessary abbreviations for readability sake. Could we use "Authorization" here?

Reply:
I used Auth because it's a common shorthand for Authentication and Authorization.  It's better than AuthN and AuthZ, is you ask me!

Good to keep it this way?

Reply:
ASP.NET's naming convention seems to use Authorization for all of this, mostly because we only care about authorized access at the endpoints of a service. Unlike auth providers (like AAD) which handle both authentication and authorization flows.

Reply:
Changed